develop-docs/application-architecture/feedback-architecture.mdx

```diff
@@ -263,7 +263,7 @@ filters, **skipping emails if:**
 This feature is available if you enable the following feature flags.
 
 - `organizations:user-feedback-ingest`
-- `organizations:user-feedback-ui`
+- `organizations:user-feedback-ui` (before v??)
```

Comment:
This is because in https://github.com/getsentry/sentry/pull/95195 we removed the flag.

I'm checking the version still..

Reply:
```suggestion
- `organizations:user-feedback-ui` (v25.6.2 and prior)
```